From 725e787a9676cf163715aa22acebdf4feb29fdfa Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 16 Oct 2024 13:54:14 +0200
Subject: [PATCH] arm64: dts: st: fix camera pipeline on stm32mp257f-dk

Correct imx335 regulator nodes name which should be
such as regulator-[0-9]v[0-9].

Change-Id: I10b26177778d021e212882dc95368861c799e17f
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/411741
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index 883da1609884..5010f67b54aa 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -131,7 +131,7 @@ hdmi_connector_in: endpoint {
 		};
 	};
 
-	imx335_2v9: imx335-2v9 {
+	imx335_2v9: regulator-2v9 {
 		compatible = "regulator-fixed";
 		regulator-name = "imx335-avdd";
 		regulator-min-microvolt = <2900000>;
@@ -139,7 +139,7 @@ imx335_2v9: imx335-2v9 {
 		regulator-always-on;
 	};
 
-	imx335_1v8: imx335-1v8 {
+	imx335_1v8: regulator-1v8 {
 		compatible = "regulator-fixed";
 		regulator-name = "imx335-ovdd";
 		regulator-min-microvolt = <1800000>;
@@ -147,7 +147,7 @@ imx335_1v8: imx335-1v8 {
 		regulator-always-on;
 	};
 
-	imx335_1v2: imx335-1v2 {
+	imx335_1v2: regulator-1v2 {
 		compatible = "regulator-fixed";
 		regulator-name = "imx335-dvdd";
 		regulator-min-microvolt = <1200000>;
-- 
2.50.1

